terraform-google-{{cookiecutter.module_name}}/test/make.sh

```diff
@@ -72,18 +72,13 @@ function docker() {
 
 # This function runs 'terraform validate' and 'terraform fmt'
 # against all directory paths which contain *.tf files.
-function check_terraform() {
-  set -e
-  echo "Running terraform validate"
+function check_terraform() {set -e
```

Comment:
In test/make.sh line 75:
function check_terraform() {set -e
                            ^-- SC1054: You need a space after the '{'.